index.html

```diff
@@ -1,222 +1,219 @@
-                            "store_ids": ["*"],
-                            "purchase_to": "2030-06-13T21:59:00Z",
-                            "purchase_from": "2080-06-06T22:00:00Z"
-                        },
-                        "limitation": {},
-                        "reward": { "type": "discount_item_new_price", "value": 5000 },
-                        "created_by": "DinteroTest",
-                        "created_at": "2021-06-04T10:21:18.029Z",
-                        "updated_at": "2021-06-04T10:21:18.029Z",
-                        "receipt_text": "Ukens tilbud",
-                        "type": "item",
-                        "links": [
-                            {
-                                "rel": "medium_discount_image",
-                                "href": "https://uploads-ssl.webflow.com/5f023ba9549ac22870f32943/5f1333ae5a496b0e0476975e_dintero-logo.svg"
-                            }
-                        ],
-                        "metadata": {
-                            "label": "Før: 100,-",
-                            "subtitle": "Ukens tilbud",
-                            "campaign_id": 1
-                        },
-                        "active": true
-                    }],
-                    theme: {
-                        fontSize: "14px",
-                        color: "rgba(0,0,0,1)",
-                        primary: "hotpink",
-                        secondary: "aqua",
-                        background: "#fff",
+        }).then(result => {
+            window.setTimeout(result.destroy, 50000);
+        });
+    </script>
+    <H2>Static test</h2>
+    <p>In this example the SDK is supplied with a list of discount objects directly.</p>
+    <div id="deals-container-static"></div>
+    <script type="text/javascript">
+        discounts.embed({
+            container: document.getElementById("deals-container-static"),
+            discounts: [{
+                "id": "1",
+                "name": "Testvare",
+                "description": "Beskrivelse av testvare",
+                "requirement": {
+                    "item": { "items": [{ "id": "testvare-1" }] },
+                    "customer": {},
+                    "store_ids": ["*"],
+                    "purchase_to": "2030-06-13T21:59:00Z",
+                    "purchase_from": "2080-06-06T22:00:00Z"
+                },
+                "limitation": {},
+                "reward": { "type": "discount_item_new_price", "value": 5000 },
+                "created_by": "DinteroTest",
+                "created_at": "2021-06-04T10:21:18.029Z",
+                "updated_at": "2021-06-04T10:21:18.029Z",
+                "receipt_text": "Ukens tilbud",
+                "type": "item",
+                "links": [
+                    {
+                        "rel": "medium_discount_image",
+                        "href": "https://uploads-ssl.webflow.com/5f023ba9549ac22870f32943/5f1333ae5a496b0e0476975e_dintero-logo.svg"
```

Comment:
maybe add an webshop link to the example?

Reply:
added